src/components/sections/rich-text/components/default.styles.ts

```diff
@@ -37,3 +38,18 @@ export const StyledUL = styled.ul`
   margin-bottom: 16px;
   color: ${({ theme }) => theme.colors.text.darkGrey};
 `;
+
+export const StyledOL = styled.ol`
+  list-style: decimal;
+  padding-left: 30px;
+  margin-bottom: 16px;
+  color: ${({ theme }) => theme.colors.text.darkGrey};
+`;
+
+export const StyledImage = styled(Image)`
+  display: block;
+  position: relative !important;
+  width: unset !important;
+  height: unset !important;
```

Comment:
I think it's not necessary to make this override, as it's preventing the full image from being displayed. 

https://github.com/bejamas/httptoolkit-website/assets/22168189/a24ce727-cf21-48c5-a46c-e4ef87ad0f19

```suggestion

```

Reply:
Could you check why these two images are not loading? It seems they are not in the public folder yet.

https://github.com/bejamas/httptoolkit-website/assets/22168189/b876fde8-8e3d-4025-bb8d-d5fe865c1481